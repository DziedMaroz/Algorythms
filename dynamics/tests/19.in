971823167875
308
6855
1651
874320964280
29489
203
5995985
53
5317
033649
44671
153726
8074
14
4947816212
5
74586
07437763
00738
1496
8955786
87
13002517132
123785389603
41
2709
47121
459207
04115375804
98383031380
06084
4
6755
5112
8180
251
1046
24
688514838
118391100
22498314944
559436818
9126
6
9752764
10277045
331
460850682
026130250
374141852260
115
9089339
568
18395383
865913592671
14416616
69581293
894460766
99092
8
618
18103
930
23114
5999
27040
4048
7216
1377334937
688
88451422258
89589495
034204
04439392
4
474741791144
144
2697
40337
467570018
6736
3943
871
3
7334471
0871842
5
36340046
309927443
9048917
55
6454735635
4277289
0490114
54918707426
28791667458
83130065
4420457625
5
31855708
799626207154
719398792
8
1
644457472146
29450791
446774294
327251619641
7632
502233148
66650864936
552937331272
88781699078
49199011
038175671
210
76061596
754825786
9
377
4597047
133
635
71831613
206211
74415322
32924
043801043
493515
03
97
674047104
08880798544
419572
4560247865
91643777
6959104486
04300145624
922179018
721056
876626
60529809
2
210
26576334
938529
271048
022028059
2
4414541127
94837159
39
489
18545
086206345978
2
005777
744
2458
0557702
101586
515851768369
709958522977
8331882
0196588748
302
852
190705346
7
4
807003
89
514767813
170976
18538803
85082489
90293260
14076
46182697621
4361
2092918
2867
056
0
530470612
4456745621
170405
4803856
62705
51353811
180532318
756019192225
11478
834427857510
663
689579608
583
564297976548
11284785
781866221542
4653
3617
74362530060
6342
264
4
41863
44042094
78954204065
66
2975408
7852401620
61613
959142373626
97886202711
62942
17
2184
517722
38859125
089
290
64410290
4
755
15068
6543
27553
1294
6
9076450606
906
71687
24206375660
3572046571
19125233566
964590751
545865285
114450999
5604106963
1334860
055
6726
85
162287587
20666629
05753176
46021329
358
123051
9571799160
3
75777
873
15105569
71202812
43
096
73163817068
4953
245630996469
3026654314
7
8188238839
86513128403
815702
06
72036
5769316
221549652
9168664
0667
454350
926227414
8808848
51245473742
7239988
11
710
5
30312
8243289
8
40196
876293366169
7039367059
0
05
27657100033
9038
3456684
12745
5714
57468083
040344084771
729
10075573889
78626059
02507482570
790417839
69157678
03
07295139
9
84219024715
429918
4732950
556394
056
96084418
