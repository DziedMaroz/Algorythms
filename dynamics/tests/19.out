10
97 1 8 2 3 1 6 7 87 5
